## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 508924`
Commit: `924d42b6ed8a112a1d9377d53abaca84e203a31f`

---
### `aarch64-darwin`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.compressed-tensors</li>
    <li>python313Packages.compressed-tensors.dist</li>
    <li>python314Packages.compressed-tensors</li>
    <li>python314Packages.compressed-tensors.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.compressed-tensors</summary>
<pre>=============================== warnings summary ===============================
../../../../../store/nvqickglpw3bfg3816744hf3hm1zd8qs-python3.13-torch-2.11.0/lib/python3.13/site-packages/torch/jit/_script.py:365: 14 warnings
  /nix/store/nvqickglpw3bfg3816744hf3hm1zd8qs-python3.13-torch-2.11.0/lib/python3.13/site-packages/torch/jit/_script.py:365: DeprecationWarning: `torch.jit.script_method` is deprecated. Please switch to `torch.compile` or `torch.export`.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module[cpu-NVFP4A16-nvfp4-pack-quantized-None] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/6xs7cggwr1cbala1gs2pjzdg9n/torchinductor__nixbld10/3b/c3bsxikyzchkr3lxjlue3ku5abb5wa5fwaab44ggbcfhgf57soej.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module[cpu-NVFP4-nvfp4-pack-quantized-None] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/6xs7cggwr1cbala1gs2pjzdg9n/torchinductor__nixbld10/3b/c3bsxikyzchkr3lxjlue3ku5abb5wa5fwaab44ggbcfhgf57soej.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module_mxfp4[cpu-MXFP4A16-mxfp4-pack-quantized] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/6xs7cggwr1cbala1gs2pjzdg9n/torchinductor__nixbld10/2n/c2ntn7igw7pwkxmh5wwopoa2rzfjb7hlvy54mf3czdldvqxbe2pn.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module_mxfp4[cpu-MXFP4-mxfp4-pack-quantized] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/6xs7cggwr1cbala1gs2pjzdg9n/torchinductor__nixbld10/2n/c2ntn7igw7pwkxmh5wwopoa2rzfjb7hlvy54mf3czdldvqxbe2pn.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
= 4 failed, 512 passed, 190 skipped, 41 deselected, 2 xfailed, 14 warnings in 15.65s =</pre>
</details>
<details>
<summary>python314Packages.compressed-tensors</summary>
<pre>=============================== warnings summary ===============================
../../../../../store/q41g634dys6kr1h9112a8rh22wg6p12z-python3.14-torch-2.11.0/lib/python3.14/site-packages/torch/jit/_script.py:359: 14 warnings
  /nix/store/q41g634dys6kr1h9112a8rh22wg6p12z-python3.14-torch-2.11.0/lib/python3.14/site-packages/torch/jit/_script.py:359: DeprecationWarning: `torch.jit.script_method` is not supported in Python 3.14+ and may break. Please switch to `torch.compile` or `torch.export`.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module[cpu-NVFP4A16-nvfp4-pack-quantized-None] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/0p9nisb8694cri6vajwk6a4y4b/torchinductor__nixbld1/e4/ce43mobem57ce2xq636scxl5l7odmbumh3vx2q6kqoonmnrsdtpw.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module[cpu-NVFP4-nvfp4-pack-quantized-None] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/0p9nisb8694cri6vajwk6a4y4b/torchinductor__nixbld1/e4/ce43mobem57ce2xq636scxl5l7odmbumh3vx2q6kqoonmnrsdtpw.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module_mxfp4[cpu-MXFP4A16-mxfp4-pack-quantized] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/0p9nisb8694cri6vajwk6a4y4b/torchinductor__nixbld1/wn/cwnoawe7n3xkwjpjhjqnyniubh77ruejqrqvt3dvncr74olhjiz7.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
FAILED tests/test_compressors/test_compress_decompress_module.py::test_compress_decompress_module_mxfp4[cpu-MXFP4-mxfp4-pack-quantized] - torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/b/0p9nisb8694cri6vajwk6a4y4b/torchinductor__nixbld1/wn/cwnoawe7n3xkwjpjhjqnyniubh77ruejqrqvt3dvncr74olhjiz7.main.so, 0x0002): symbol not found in flat namespace &#x27;___kmpc_barrier&#x27;

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you&#x27;re reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS=&quot;+dynamo&quot;
= 4 failed, 512 passed, 190 skipped, 41 deselected, 2 xfailed, 14 warnings in 15.63s =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/compressed-tensors/default.nix
+++ b/pkgs/development/python-modules/compressed-tensors/default.nix
@@ -3,7 +3,6 @@
   stdenv,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonAtLeast,
 
   # build-system
   setuptools,
@@ -25,7 +24,7 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "compressed-tensors";
-  version = "0.14.0.1";
+  version = "0.15.0.1";
   pyproject = true;
 
   # Release on PyPI is missing the `utils` directory, which `setup.py` wants to import
@@ -33,7 +32,7 @@ buildPythonPackage (finalAttrs: {
     owner = "neuralmagic";
     repo = "compressed-tensors";
     tag = finalAttrs.version;
-    hash = "sha256-bncglC6qRE4D6rjoNOXRKh+7ZMSTc9zc3NVzA+NaXuE=";
+    hash = "sha256-iiYo3Vne2CYlj+wMHxQFTTU7gb8oNwPtCe873nX5KgA=";
   };
 
   postPatch = ''
@@ -46,7 +45,9 @@ buildPythonPackage (finalAttrs: {
     setuptools-scm
   ];
 
-  buildInputs = lib.optional stdenv.cc.isClang llvmPackages.openmp;
+  buildInputs = lib.optionals stdenv.cc.isClang [
+    llvmPackages.openmp
+  ];
 
   dependencies = [
     frozendict
@@ -55,10 +56,6 @@ buildPythonPackage (finalAttrs: {
     torch
     transformers
   ];
-
-  pythonRelaxDeps = [ "transformers" ];
-
-  doCheck = true;
 
   pythonImportsCheck = [ "compressed_tensors" ];
 
@@ -89,6 +86,11 @@ buildPythonPackage (finalAttrs: {
     "test_set_item"
     "test_set_item_buffers"
     "test_update_offload_parameter_with_grad"
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    # torch._inductor.exc.InductorError: ImportError: dlopen(/nix/var/nix/builds/nix-25002-542173852/torchinductor__nixbld1/xo/cxovsevcfanmw7lgoddbnyhoxes3nzlu7ecugxedaq2zr4f6b2qh.main.so, 0x0002):
+    # symbol not found in flat namespace '___kmpc_barrier'
+    "test_compress_decompress_module"
   ];
 
   disabledTestPaths = [

